manual/manual/tutorials/coreexamples.etex

```diff
@@ -110,11 +111,16 @@ sort [3.14; 2.718];;
 The "sort" function above does not modify its input list: it builds
 and returns a new list containing the same elements as the input list,
 in ascending order. There is actually no way in OCaml to modify
-in-place a list once it is built: we say that lists are  {\em immutable}
+a list in place once it is built: we say that lists are  {\em immutable}
 data structures. Most OCaml data structures are immutable, but a few
 (most notably arrays) are {\em mutable}, meaning that they can be
 modified in-place at any time.
 
+The OCaml notation for the type of a function with multiple arguments is
+"arg1_type -> arg2_type -> ... -> return_type".  For example,
+the type inferred for insert, "'a -> 'a list -> 'a list", means that insert
+takes two arguments, an element of any type "'a" and a list of elements of
+the same type "'a" and returns a list of the same type.
```

Comment:
@Octachron - does the style of the manual does require some kind of formatting when referring to the `insert` function, too?

(text looks great, though!)

Reply:
I missed this point: Using `"insert"` when refering to the function named `insert` would be better (but not strictly required).